src/Elmish/Hooks/UseEffect.purs

```diff
@@ -26,24 +28,68 @@ import Elmish.Hooks.Type (Hook, mkHook, uniqueNameFromCurrentCallStack, uniqueNa
 -- |   pure $ H.fragment $ todoView <$> todos
 -- | ```
 useEffect :: Aff Unit -> Hook Unit
-useEffect = useEffect' identity uniqueNameFromCurrentCallStack
+useEffect aff =
+  useEffect_ name unit $ const aff
+  where
+    name = uniqueNameFromCurrentCallStack { skipFrames: 3 }
```

Comment:
Why 3?

Reply:
Did you mean to have this `name` bound in `useEffect_` instead?

Reply:
I’ll have to look at the trace again, but for some reason this seems to be the right number.